addons/website_mass_mailing/models/mailing_contact.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models
+
+
+class MassMailingContact(models.Model):
+    _inherit = 'mailing.contact'
+
+    @api.model
+    def _get_searchable_fields(self):
```

Comment:
That's a bit vague naming, and not sure it really belongs to the model. If we keep the contact model update, can't we have a small tool controller method overriden (like other sub methods) ?